icu4j/main/classes/core/src/com/ibm/icu/impl/number/LongNameHandler.java

```diff
@@ -79,7 +79,7 @@ public void put(UResource.Key key, UResource.Value value, boolean noFallback) {
 
     // NOTE: outArray MUST have at least ARRAY_LENGTH entries. No bounds checking is performed.
 
-    private static void getMeasureData(
+    protected static void getMeasureData(
```

Comment:
This function is "protected" rather than "private" because MixedUnitLongNameHandler is referencing it. It apparently doesn't need to be public.